modules/ols-0-1-5-release-notes.adoc

```diff
@@ -0,0 +1,18 @@
+// This module is used in the following assemblies:
+// release_notes/ols-release-notes.adoc
+
+:_mod-docs-content-type: Reference
+[id="ols-0-1-5-release-notes_{context}"]
+= {ols-long} version 0.1.5
+
+{ols-offical} 0.1.5 is now available on {ocp-product-title} 4.15 or later.
+
+== New features
+
+This release introduces the following functionality:
+
+* This release introduces support for Large Language Model (LLM) authentication using {azure-official} with Entra ID.
```

Comment:
Since it's 'technically' Microsoft Entra ID now, do we want to say: 

```suggestion
* This release introduces support for Large Language Model (LLM) authentication using {azure-official} with Microsoft Entra ID.
```

Which is a little redundant with Microsoft being used x2 so close together.

Reply:
created attribute for entra that displays as Microsoft Entra ID.